engines/private/module.mk

```diff
@@ -0,0 +1,25 @@
+MODULE := engines/private
+ 
+MODULE_OBJS := \
+    code.o \
+    cursors.o \
+    funcs.o \
+    grammar.o \
+	lexer.o \
+	metaengine.o \
+    private.o \
+	symbol.o
+ 
+MODULE_DIRS += \
+    engines/private
+ 
+# This module can be built as a plugin
+ifeq ($(ENABLE_PRIVATE), DYNAMIC_PLUGIN)
+PLUGIN := 1
+endif
+ 
+# Include common rules 
+include $(srcdir)/rules.mk
```

Comment:
I recommend putting here a target for generating the flex/bison output. See here: https://github.com/scummvm/scummvm/blob/master/engines/director/module.mk#L43

Reply:
Done.